src/components/pages/Categories.js

```diff
@@ -1,5 +1,24 @@
-import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../../redux/categories/categories';
 
-const Categories = () => <button type="button">Check status</button>;
+const Categories = () => {
+  const dispatch = useDispatch();
 
+  const handleClick = () => {
+    const notice = { text: 'Under construction' };
+    dispatch(checkStatus(notice));
+  };
+  const statusCheck = useSelector((state) => state.Categories);
+
+  return (
+    <div className="">
+      <button type="button" className="check-status" onClick={handleClick}>
+        check status
+      </button>
```

Comment:
- [ ] Clicking on the Categories nav link navigates to a blank page, with the attached error thrown in the console, as can be seen below. Please ensure that your check status button is properly implemented and should display the message 'Under construction when clicked.

**Categories Page Error:**
![image](https://user-images.githubusercontent.com/12745474/214682758-40a0dda3-9f06-4aa2-8f57-80800e3ecd65.png)

![image](https://user-images.githubusercontent.com/12745474/214681818-2d5040bd-6cb1-4a95-bdbd-c63629e0fcf1.png)

Reply:
Thank you for your suggestions -  I have fixed the category link and the readme.
![Screenshot 2023-01-26 at 12 11 38 PM](https://user-images.githubusercontent.com/106531695/214800027-9abfd670-81d1-4a95-839b-e9f8692022a4.png)